nextflow.config

```
@@ -13,6 +13,7 @@
     // Input options
     input                      = null
     fasta                      = null
+    queries_dir                = 'assets/queries/'
```

```suggestion
    queries_dir                = "${projectDir}/assets/queries/"
```
to ensure that this will work no matter where you execute the pipeline